Code/CoreData/RKRelationshipConnectionOperation.m

```diff
@@ -48,11 +48,13 @@ static BOOL RKConnectionAttributeValuesIsNotConnectable(NSDictionary *attributeV
 {
     NSCAssert([connection isForeignKeyConnection], @"Only valid for a foreign key connection");
     NSMutableDictionary *destinationEntityAttributeValues = [NSMutableDictionary dictionaryWithCapacity:[connection.attributes count]];
-    for (NSString *sourceAttribute in connection.attributes) {
-        NSString *destinationAttribute = (connection.attributes)[sourceAttribute];
-        id sourceValue = [managedObject valueForKey:sourceAttribute];
-        [destinationEntityAttributeValues setValue:sourceValue ?: [NSNull null] forKey:destinationAttribute];
-    }
+    [managedObject.managedObjectContext performBlockAndWait:^{
```

Comment:
If `managedObject` has no context then `destinationEntityAttributeValues` is empty, but it pass final check. After that it will crash at line 160 because `if` statement at line 156 also mistakenly passed. I think #2217 somehow fix that, but it potentially danger approach. @jeffblake @cfis @segiddins